cookbooks/bcpc-hadoop/recipes/hive_hcatalog.rb

```diff
@@ -37,34 +37,25 @@
   hdfs dfs -chmod 1777 /user/hive
   hdfs dfs -chown hive:hdfs /user/hive
   EOH
-  user "hdfs"
+  user 'hdfs'
 end
 
-bash "create-hive-warehouse" do
+bash 'create-hive-warehouse' do
   code <<-EOH
-  hdfs dfs -mkdir -p /apps/hive/warehousehadoop
-  hdfs dfs -chmod -R 775 /apps/hive
-  hdfs dfs -chown -R hive:hdfs /apps/hive
+  hdfs dfs -mkdir -p #{node['bcpc']['hadoop']['hive']['warehouse']['dir']}
+  hdfs dfs -chmod -R 775 #{node['bcpc']['hadoop']['hive']['warehouse']['dir']}
+  hdfs dfs -chown -R hive:hdfs #{node['bcpc']['hadoop']['hive']['warehouse']['dir']}
```

Comment:
I see you changed attributes to `:` notation [here](https://github.com/bloomberg/chef-bach/pull/750/files#diff-3fb9d1f9c1a180d2ea11310174864fccR43). Why not follow the same pattern here?